### ISSUE TITLE
Issue with opening https://developers.nativedocuments.com/

### ISSUE THREAD

**Issue by shwetasinghgosal** [NativeDocuments/docx-to-pdf-on-AWS-Lambda#10]
I am not able to access ND_DEV_ID, ND_DEV_SECRET from https://developers.nativedocuments.com/. Can anyone here help me please?

**Comment by Lolispo** [comment]
Yes, The website is not opening, would be good to get some information why it's down or an alternative way to receive the key information
